backend/src/main/java/com/github/devbk84/backend/controller/ProductController.java

```diff
@@ -0,0 +1,44 @@
+package com.github.devbk84.backend.controller;
+
+import com.github.devbk84.backend.models.Product;
+import com.github.devbk84.backend.service.ProductService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/products")
+public class ProductController {
+
+    private final ProductService productService;
+
+    public ProductController(ProductService productService) {
+
+        this.productService = productService;
+    }
+
+    @GetMapping
+    public List<Product> getAllProducts(){
+
+        return productService.getAllProducts();
+    }
+
+    @GetMapping(path = "/{id}")
+    public Product getProductsById(@PathVariable String id){
+        return productService.getProductsById(id);
```

Comment:
You just recive one product, you should rename the methods to getProductById 
```suggestion
    public Product getProductById(@PathVariable String id){
        return productService.getProductById(id);
```